src/pages/ServiceList/ServiceListComponent.tsx

```diff
@@ -188,6 +189,14 @@ class ServiceListComponent extends ListComponent.Component<
     const pageStart = (this.state.pagination.page - 1) * this.state.pagination.perPage;
     let pageEnd = pageStart + this.state.pagination.perPage;
     pageEnd = pageEnd < this.state.listItems.length ? pageEnd : this.state.listItems.length;
+    let hasApiColumn = false;
+    for (let i = pageStart; i < pageEnd; i++) {
```

Comment:
`hasApiColumn = this.state.listItems.slice(pageStart, pageEnd).some(item => item.apiType)`, maybe?

Reply:
Wooo, I will try that :-) Thanks